08-communicating-results.Rmd

```diff
@@ -170,7 +170,7 @@ anes_des <- anes_in %>%
   )
 ```
 
-Next, we select our desired data points.
+Next, we select our desired data points. 
```

Comment:
I think we need to do a better job of walking the reader through this.  Let's break up the code and narrate through it.

Reply:
Also this is the same graph that is created in chapter 7.  Can we use a different variable, or RECS data instead?